### PR TITLE
Updated link to manual

### PR DESCRIPTION
**Pull request by sourabhtk37**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,5 +79,5 @@ terminology, etc. is wellcome.
 
 For more details see the wiki pages and the manual page:
  - https://github.com/dashohoxha/egpg/wiki
- - http://dashohoxha.github.io/egpg/gnupg-2.1/man/
+ - http://dashohoxha.fs.al/easy-gnupg-presentation/
```
Comment (dashohoxha) [inline README.md]: The correct link is: https://easygnupg.github.io/egpg/gnupg-2.1/man/

Comment (dashohoxha) [reply README.md]: Also the link for the /gnupg-2.0/man/ should be fixed.

Comment (dashohoxha) [reply README.md]: Basically, the domain `dashohoxha.github.io` should be replaced by `easygnupg.github.io` everywhere. It needs a find/replace to all files. I don't know which editor you are using, but in Emacs this is an easy task.

Comment (sourabhtk37) [reply README.md]: Yes, using sed too.

**Review by dashohoxha — APPROVED** [review]
I gues this is ok.
The other occurrences are on the other branches (gnupg-2.0 and gh-pages).

**Pull request merged**